packages/civic-sandbox/src/components/App/index.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import { css } from 'emotion';
+import Packages from '../Packages';
```

Comment:
what this does this do? is this necessary? @DingoEatingFuzz

Reply:
I should probably be clearer with my naming of that component. I named the component for the sandbox package selector `Packages`. Should probably name it `PackageSelector` or `SandboxPackageSelector`